src/net_processing.cpp

```diff
@@ -3506,7 +3506,6 @@ bool ProcessMessage(CNode& pfrom, const std::string& msg_type, CDataStream& vRec
         vRecv >> newFeeFilter;
         if (MoneyRange(newFeeFilter)) {
             if (pfrom.m_tx_relay != nullptr) {
-                LOCK(pfrom.m_tx_relay->cs_feeFilter);
                 pfrom.m_tx_relay->minFeeFilter = newFeeFilter;
```

Comment:
nit: Make atomic operations explicit:
```suggestion
                pfrom.m_tx_relay->minFeeFilter.store(newFeeFilter);
```

Reply:
Going to leave as is to minimize the diff